docs/rotate-dev-certificates.md

```diff
@@ -0,0 +1,57 @@
+# Rotate Certificates on Dev Subscription
+
+Once every year, we need to rotate certificates on dev subscription. At the time of writing this, new certificates were generated on <u>Jan 5th, 2022</u>. This document goes over the steps taken to rotate the certificates.
+
+1. Generate new certificates
+
+```bash
+go run ./hack/genkey -client firstparty
+mv firstparty.* secrets
+
+go run ./hack/genkey -ca vpn-ca
+mv vpn-ca.* secrets
+
+go run ./hack/genkey -client -keyFile secrets/vpn-ca.key -certFile secrets/vpn-ca.crt vpn-client
+mv vpn-client.* secrets
+
+go run ./hack/genkey proxy
+mv proxy.* secrets
+
+go run ./hack/genkey -client proxy-client
+mv proxy-client.* secrets
+
+ssh-keygen -f secrets/proxy_id_rsa -N ''
+
+go run ./hack/genkey localhost
+mv localhost.* secrets
+```
+
+2. Run import_certs_secrets. This will import certificates to keyvault and then set the secrets.
+
+```bash
+source ./hack/devtools/deploy-shared-env.sh
+import_certs_secrets
+```
+
+3. The OpenVPN configuration file needs to be updated to enable tunneling to the vpn. To achieve this, edit the `vpn-<region>.ovpn` file in secrets and add the `vpn-client certificate` and `vpn-client certificate private key`.
```

Comment:
This may change with the way we create the ovpn config file as it can reference a file for a key/cert.  I'm fine with it now, but may want to update it later.